Makefile

```
@@ -125,12 +125,13 @@ namespace:
 deploy: certs
 	@{ \
 	set -e ;\
-	TEMP_FILE=/tmp/authorino-deploy-$$(openssl rand -hex 4).yaml ;\
+	TEMP_DIR=$(PROJECT_DIR)/tmp ;\
+	mkdir $$TEMP_DIR ;\
```

```suggestion
	mkdir -p $$TEMP_DIR ;\
```

otherwise it can fail with `File exists`.